tests/stress_tests/common/utils.cpp

```diff
@@ -25,11 +25,12 @@ std::string fileNameNoExt(const std::string &filepath) {
 
 
 static size_t parseLine(char* line) {
-    // This assumes that a digit will be found and the line ends in " Kb".
     size_t i = strlen(line);
     const char* p = line;
     while (*p <'0' || *p > '9') p++;
```

Comment:
could we add a check pointer p is still inside string?

Reply:
Done